velox/expression/EvalCtx.cpp

```diff
@@ -219,11 +219,13 @@ void EvalCtx::convertElementErrorsToTopLevelNulls(
     return;
   }
 
+  auto rawNulls = result->mutableRawNulls();
```

Comment:
In case there will be not errors, will we end up allocating nulls buffer needlessly?